sycl/include/sycl/ext/oneapi/experimental/bfloat16.hpp

```diff
@@ -14,10 +14,10 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace ext {
-namespace intel {
+namespace oneapi {
 namespace experimental {
 
-class [[sycl_detail::uses_aspects(ext_intel_bf16_conversion)]] bfloat16 {
+class bfloat16 {
```

Comment:
Just curious, did you remove this C++ attribute intentionally?

Removing it is probably not a big problem because we don't have support in the rest of the tool chain yet to take advantage of this attribute.  If there's no reason to remove it, though, it seems better to keep it.

Reply:
Yes it was intentional.  I think that it will be replaced by the oneapi aspect in https://github.com/intel/llvm/pull/5720.

The only reason for deleting it is that the cuda backend does not have this ext_intel_bf16_conversion aspect (which as I understand it will be replaced by the oneapi aspect generally).  However I can try adding it back and seeing if the cuda backend bfloat16 test passes with it included, since I'm not sure how it will behave. Shall I do this? If the cuda backend test passes then this line can remain unchanged if that is preferred.

Reply:
> Oh, I see. I thought the aspect was renamed in this PR, but I see now that it is not.
> 
> When #5720 is merged, we should add this C++ attribute back and use the new aspect. 

So is it OK to leave it as it is? I would remember to add it back when the new aspect is available....

> I think you could probably keep the attribute in this PR with the old aspect name, and it would probably not hurt CUDA. The attribute will cause the front-end to put some metadata on the IR, but I think all IR passes ignore that metadata right now.

Or should I add it back now with the old aspect name?

Reply:
Either way is OK.